lib/matplotlib/__init__.py

```diff
@@ -105,6 +105,18 @@
 
 import six
 
+import sys
+if sys.version_info < (3, 5):  # noqa: E402
+    raise ImportError("""
+Matplotlib 3.0+ does not support Python 2.x, 3.0, 3.1, 3.2, 3.3, or 3.4.
+Beginning with Matplotlib 3.0, Python 3.5 and above is required.
+
+See Matplotlib `INSTALL.rst` file for more information:
```

Comment:
For more information see https://github.com/matplotlib/matplotlib/blob/master/INSTALL.rst

Reply:
Doesn't fit on a line nicely.